SU2_CFD/include/variables/CAdjEulerVariable.hpp

```diff
@@ -0,0 +1,150 @@
+/*!
+ * \file CAdjEulerVariable.hpp
```

Comment:
Looks like you are already doing it, but this is a nice opportunity to fix any indentation / white space / line ending issues when creating these new files. Can you please double check?

Reply:
I think indentation was already ok, I striped all trailing white spaces now.
There are also many (many...) comments that were copied and do not match the function, that is not a huge deal since these methods explain themselves. I fixed some but I will not fix all of them...

Reply:
Certainly not expected that you fix them all.. let's get some help on that in a later sweep once merged.

Thanks for the clean up! Can you also verify that all line endings are linux style and not windows format?

Reply:
Apparently everything was LF-ended already, at least the IDE I use did not pick up anything.